### DISCUSSION TITLE
Complete Noob - Not sure how to proceed (Marlin Firmware)

### DISCUSSION THREAD

**Discussion by BartImler** [community/community#64154] [New to GitHub]
Hi all,

I'm completely new to GitHub.  I came here to get an example configuration from Marlin Firmware for Ender 3 Neo.  There is an example config in the import.2.1.x branch, (I think I am using the correct terminology, maybe not), https://github.com/MarlinFirmware/Configurations/tree/import-2.1.x/config/examples/Creality/Ender-3%20Neo   but not in the 2.1.2.1 Release branch.  https://github.com/MarlinFirmware/Configurations/tree/release-2.1.2.1/config/examples/Creality

I tried to compile the import version configuration.h file, but it reportedly has some hard coded errors built in directing me to use the correct release version. (which doesn't exist yet).

From what I can make out, the correct protocol would be to submit a pull request to add the import file into the release branch, but I have no clue how to do that.  I ended up on a page telling me to compare, but I don't know what I am doing or what I am supposed to compare (or how that translates into a pull request).  Given that I did not create the import version file, I'm not even sure if I am allowed to submit such a request.

In another world I might try to contact the repository "owner" (thinkyhead) , but I'm not sure if that is correct or acceptable protocol here (or if it is even possible)

Any guidance would be most appreciated.

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
